<launch>
	<param name="Alpha_Max" value="45"/>
	<param name="Alpha_Min" value="-45"/>
	<param name="d" value="0.9"/>
	<param name="r" value="0.7"/>
	
<node pkg="joy" type="joy_node" name="joy_node" respawn="true" />
	
<node pkg="controls" type="encoders" name="encoders" respawn="true"/>
	
<node pkg="controls" type="overriding_node" name = "overriding_layer" respawn="true" output="screen"/>
	
<node pkg="controls" type="vxpid_node" name="vxpid_node" respawn="true">
	<param name="Kp_Vx" value="10"/>
	<param name="Ki_Vx" value="5"/>
	<param name="Kd_Vx" value="0.02"/>
	<param name="PWM_min" value="15"/>
	<param name="PWM_max" value="85"/>
	<param name="PWM_PERIOD_TIME" value="50000000"/>
</node>

<node pkg="controls" type="wpid_node" name="wpid_node" respawn="true">
	<param name="Kp_W" value="10"/>
	<param name="Ki_W" value="5"/>
	<param name="Kd_W" value="0.02"/>
</node>
	
</launch>
 
